doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -78,145 +78,107 @@ Displaying
 
+   Sets :token:`flag` off. Scoping qualifiers are
+   described :ref:`here <set_unset_scope_qualifiers>`.
 
 .. cmd:: Test @flag
 
-   This command prints whether :n:`@flag` is on or off.
-
-
-.. cmd:: Set @option @value
+   Prints the current value of :token:`flag`.
 
-   This command sets :n:`@option` to :n:`@value`. The original value of ` option` is
-   restored when the current module ends.
 
-   .. TODO : option and value are not syntax entries
+.. cmd:: {? Local | Global | Export } Set @option {| @num | @string }
+   :name: Set @option
 
-   .. cmdv:: Local Set @option @value
+   Sets :token:`option` to the specified value.  Scoping qualifiers are
+   described :ref:`here <set_unset_scope_qualifiers>`.
 
-      This command sets :n:`@option` to :n:`@value`. The
-      original value of :n:`@option` is restored at the end of the module.
+.. cmd:: {? Local | Global | Export } Unset @option
+   :name: Unset @option
 
-   .. cmdv:: Global Set @option @value
+   Sets :token:`option` to its default value.  Scoping qualifiers are
+   described :ref:`here <set_unset_scope_qualifiers>`.
 
-      This command sets :n:`@option` to :n:`@value`. The
-      original value of :n:`@option` is *not* restored at the end of the module.
-      Additionally, if set in a file, :n:`@option` is set to value when the file
-      is `Require`-d.
-
-   .. cmdv:: Export Set @option
-
-      This command set :n:`@option` to :n:`@value`. The original state
-      of :n:`@option` is restored at the end of the current module, but :n:`@option`
-      is set to :n:`@value` when this module is imported.
-
-
-.. cmd:: Unset @option
-
-   This command turns off :n:`@option`.
-
-   .. cmdv:: Local Unset @option
+.. cmd:: Test @option
 
-      This command turns off :n:`@option`. The original state of :n:`@option`
-      is restored when the current *section* ends.
+   Prints the current value of :token:`option`.
 
-   .. cmdv:: Global Unset @option
+.. cmd:: Print Options
 
-      This command turns off :n:`@option`. The original state of :n:`@option`
-      is *not* restored at the end of the module. Additionally, if unset in a file,
-      :n:`@option` is reset to its default value when the file is `Require`-d.
+   Prints the current value of all flags and options, and the names of all tables.
 
-   .. cmdv:: Export Unset @option
 
-      This command turns off :n:`@option`. The original state of :n:`@option`
-      is restored at the end of the current module, but :n:`@option` is set to
-      its default value when this module is imported.
+.. cmd:: Add @table @string
+   :name: Add `table` `value`
 
+   Adds :token:`string` to :token:`table`.
 
-.. cmd:: Test @option
+.. cmd:: Remove @table @string
+   :name: Remove @table @string
 
-   This command prints the current value of :n:`@option`.
+   Removes :token:`string` from :token:`table`.
 
+.. comment THIS DIDN'T WORK, e.g. for "Test Search Blacklist "foo"."
+   cmd:: Test @table @string
+   :name: Test @table @string
```

Comment:
This syntax does not exist. What exists is `Test @table` which is synonymous for `Print Table @table` (and of course also `Test @table for @string` and `Test @table for @qualid`).

Reply:
I notice that `Add Printing Coercion` is documented as taking a `qualid`, while the other `Add Printing XXX` command say they take `ident`s.  Is that correct or are they all qualids?

Reply:
Ah, well spotted. They indeed all take a qualid.

Reply:
FWIW, I see there is an `Unset Printing Records` command, but none of the other tables support this.

Reply:
Actually, `Printing Records` is not a table, so there is a bug in the example given in the manual.  It could be instead ``... or tables (e.g. :cmd:`Add Printing Let`)``.